enterprise/cluster/src/main/java/org/neo4j/cluster/protocol/atomicbroadcast/multipaxos/InMemoryAcceptorInstanceStore.java

```diff
@@ -73,7 +73,7 @@ public void lastDelivered( InstanceId instanceId )
     {
         for ( long i = lastDeliveredInstanceId; i <= instanceId.getId(); i++ )
```

Comment:
Better remove this all together or perhaps replace it with a comment explaining what used to happen here to maintain a more accessible history?